File: engine/engine-model/src/main/scala/net/server/channel/packet/npc/NPCTalkPacket.scala
package net.server.channel.packet.npc

class NPCTalkPacket(private var _available: Int, private var _first: Int, private var _second: Byte, private var _third: Byte) extends BaseNPCAnimationPacket(_available = _available) {
  def first: Int = _first

  def second: Byte = _second

  def third: Byte = _third
}

File: engine/engine-model/src/main/java/net/server/channel/packet/npc/NPCTalkPacket.java
package net.server.channel.packet.npc;

public class NPCTalkPacket extends BaseNPCAnimationPacket {
   private final Integer first;

   private final Byte second;

   private final Byte third;

   public NPCTalkPacket(Integer available, Integer first, Byte second, Byte third) {
      super(available);
      this.first = first;
      this.second = second;
      this.third = third;
   }

   public Integer first() {
      return first;
   }

   public Byte second() {
      return second;
   }

   public Byte third() {
      return third;
   }
}
